src/System.Runtime.Intrinsics.X86/src/System/Runtime/Intrinsics/X86/Avx2.cs

```diff
@@ -0,0 +1,725 @@
+
+        // __m256i _mm256_max_epi8 (__m256i a, __m256i b)
+        public static Vector256<sbyte> Max(Vector256<sbyte> left, Vector256<sbyte> right) { throw new NotImplementedException(); }
+        // __m256i _mm256_max_epu8 (__m256i a, __m256i b)
+        public static Vector256<byte> Max(Vector256<byte> left, Vector256<short> right) { throw new NotImplementedException(); }
+        // __m256i _mm256_max_epi16 (__m256i a, __m256i b)
+        public static Vector256<short> Max(Vector256<short> left, Vector256<short> right) { throw new NotImplementedException(); }
+        // __m256i _mm256_max_epu16 (__m256i a, __m256i b)
+        public static Vector256<ushort> Max(Vector256<ushort> left, Vector256<ushort> right) { throw new NotImplementedException(); }
+        // __m256i _mm256_max_epi32 (__m256i a, __m256i b)
+        public static Vector256<int> Max(Vector256<int> left, Vector256<int> right) { throw new NotImplementedException(); }
+        // __m256i _mm256_max_epu32 (__m256i a, __m256i b)
+        public static Vector256<uint> Max(Vector256<uint> left, Vector256<uint> right) { throw new NotImplementedException(); }
+        
+        // __m256i _mm256_min_epi8 (__m256i a, __m256i b)
+        public static Vector256<sbyte> Min(Vector256<sbyte> left, Vector256<sbyte> right) { throw new NotImplementedException(); }
+        // __m256i _mm256_min_epu8 (__m256i a, __m256i b)
+        public static Vector256<byte> Min(Vector256<byte> left, Vector256<byte> right) { throw new NotImplementedException(); }
+        // __m256i _mm256_min_epi16 (__m256i a, __m256i b)
+        public static Vector256<short> Min(Vector256<short> left, Vector256<short> right) { throw new NotImplementedException(); }
+        // __m256i _mm256_min_epu16 (__m256i a, __m256i b)
+        public static Vector256<ushort> Min(Vector256<ushort> left, Vector256<ushort> right) { throw new NotImplementedException(); }
+        // __m256i _mm256_min_epi32 (__m256i a, __m256i b)
+        public static Vector256<int> Min(Vector256<int> left, Vector256<int> right) { throw new NotImplementedException(); }
+        // __m256i _mm256_min_epu32 (__m256i a, __m256i b)
+        public static Vector256<uint> Min(Vector256<uint> left, Vector256<uint> right) { throw new NotImplementedException(); }
+        
+        // int _mm256_movemask_epi8 (__m256i a)
+        public static int MoveMask(Vector256<sbyte> value) { throw new NotImplementedException(); }
+        // int _mm256_movemask_epi8 (__m256i a)
+        public static int MoveMask(Vector256<byte> value) { throw new NotImplementedException(); }
+
+        // __m256i _mm256_mpsadbw_epu8 (__m256i a, __m256i b, const int imm8)
+        public static Vector256<ushort> MultipleSumAbsoluteDifference(Vector256<byte> left, Vector256<byte> right, byte mask) { throw new NotImplementedException(); }
+                
```

Comment:
Perhaps `MultipleSumsOfAbsoluteDifferences` what exactly communicates action of method